src/main/java/group1/mips_simulator/components/instructionParts/RXIA_Instruction.java

```diff
@@ -0,0 +1,58 @@
+package group1.mips_simulator.components.instructionParts;
+
+import java.util.Vector;
+
```

Comment:
just wondering if theres any cases of non-RXIA Instructions in our code? we might be able to elevate these to Instructions? Unless this is to account for things like LOC and Data which arent *particularly* defined lol

either way no change needed, just wanted to voice the thought

Reply:
Discussed in class.